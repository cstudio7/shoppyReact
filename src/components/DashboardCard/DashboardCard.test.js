import React from 'react';
import { shallow } from 'enzyme';
import DasboardCard from './index';

describe('Test the DasboardCard Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<DasboardCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
