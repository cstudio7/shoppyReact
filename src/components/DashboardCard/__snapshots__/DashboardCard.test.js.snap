// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the DasboardCard Component should match snapshot 1`] = `
<Fragment>
  <div
    className="card"
    key="0"
  >
    <div
      className="card__side card__side--front card__side--front-1"
    >
      <div
        className="card__description"
      >
        Categories
      </div>
      <Card
        className="infoCard"
      >
        <h1
          className="numberOfItems"
        >
          0
        </h1>
        <Link
          replace={false}
          to="/categories"
        >
          <Button
            animated={true}
            as="button"
          >
            <ButtonContent
              visible={true}
            >
              View All
            </ButtonContent>
            <ButtonContent
              hidden={true}
            >
              <Icon
                as="i"
                name="arrow right"
              />
            </ButtonContent>
          </Button>
        </Link>
      </Card>
    </div>
  </div>
  <div
    className="card"
    key="1"
  >
    <div
      className="card__side card__side--front card__side--front-2"
    >
      <div
        className="card__description"
      >
        Products
      </div>
      <Card
        className="infoCard"
      >
        <h1
          className="numberOfItems"
        >
          0
        </h1>
        <Link
          replace={false}
          to="/products"
        >
          <Button
            animated={true}
            as="button"
          >
            <ButtonContent
              visible={true}
            >
              View All
            </ButtonContent>
            <ButtonContent
              hidden={true}
            >
              <Icon
                as="i"
                name="arrow right"
              />
            </ButtonContent>
          </Button>
        </Link>
      </Card>
    </div>
  </div>
</Fragment>
`;

exports[`Test the DasboardCard Component should match snapshot 2`] = `
<Fragment>
  <div
    className="card"
    key="0"
  >
    <div
      className="card__side card__side--front card__side--front-1"
    >
      <div
        className="card__description"
      >
        Categories
      </div>
      <Card
        className="infoCard"
      >
        <h1
          className="numberOfItems"
        >
          3
        </h1>
        <Link
          replace={false}
          to="/categories"
        >
          <Button
            animated={true}
            as="button"
          >
            <ButtonContent
              visible={true}
            >
              View All
            </ButtonContent>
            <ButtonContent
              hidden={true}
            >
              <Icon
                as="i"
                name="arrow right"
              />
            </ButtonContent>
          </Button>
        </Link>
      </Card>
    </div>
  </div>
  <div
    className="card"
    key="1"
  >
    <div
      className="card__side card__side--front card__side--front-2"
    >
      <div
        className="card__description"
      >
        Products
      </div>
      <Card
        className="infoCard"
      >
        <h1
          className="numberOfItems"
        >
          3
        </h1>
        <Link
          replace={false}
          to="/products"
        >
          <Button
            animated={true}
            as="button"
          >
            <ButtonContent
              visible={true}
            >
              View All
            </ButtonContent>
            <ButtonContent
              hidden={true}
            >
              <Icon
                as="i"
                name="arrow right"
              />
            </ButtonContent>
          </Button>
        </Link>
      </Card>
    </div>
  </div>
  <div
    className="card"
    key="2"
  >
    <div
      className="card__side card__side--front card__side--front-3"
    >
      <div
        className="card__description"
      >
        Attendants
      </div>
      <Card
        className="infoCard"
      >
        <h1
          className="numberOfItems"
        >
          2
        </h1>
        <Link
          replace={false}
          to="/attendants"
        >
          <Button
            animated={true}
            as="button"
          >
            <ButtonContent
              visible={true}
            >
              View All
            </ButtonContent>
            <ButtonContent
              hidden={true}
            >
              <Icon
                as="i"
                name="arrow right"
              />
            </ButtonContent>
          </Button>
        </Link>
      </Card>
    </div>
  </div>
</Fragment>
`;
